docs/customization/validation.rst

```diff
@@ -11,7 +11,7 @@ Let's take the example of changing the length of ``name`` for the ``Product`` en
 In the ``sylius`` validation group the minimum length is equal to 2.
 What if you'd want to have at least 10 characters?
 
-1. Create the ``app\Resources\config\validation.yml``.
+1. Create the ``AppBundle\Resources\config\validation.yml``.
```

Comment:
First of all... to make it work, and secondly -> http://symfony.com/doc/current/validation/groups.html SYmfony's best practices place them here.